dev/fattest.simplicity/src/componenttest/rules/repeater/EE7FeatureReplacementAction.java

```diff
@@ -53,12 +53,15 @@
                                                  "mdb-3.2",
                                                  "componenttest-1.0",
                                                  "txtest-1.0",
-                                                 "websocket-1.1",
+                                                 "websocket-1.0",
```

Comment:
Why this change? The `webProfile-7.0` feature includes websocket-1.1.

Reply:
Fair point.  I will add websocket-1.0 as a special case.